content/workshops/security_containers/exercise1.9.adoc

```diff
@@ -112,7 +112,10 @@ setgid, setuid <2>
 
 == Deep Dive into Capabilities
 
-Lets look deeper into each of these capabilities.
+Lets look deeper into each of these Linux Capabilities.
```

Comment:
just a redline here, but do want that as "Linux Capabilities" or "Linux capabilities" ... lowercase 'c'